.travis.yml

```diff
@@ -4,7 +4,7 @@ before_install:
    - sudo apt-get -y install libtool automake autoconf autoconf-archive
    - sudo apt-get -y install texlive-latex-base texlive-latex-recommended
    - sudo apt-get -y install texlive-latex-extra texlive-fonts-recommended texinfo
-   - sudo apt-get -y install python-pygments python-docutils python-sphinx
+   - pip install pygments docutils sphinx
```

Comment:
Much likely this will fail without sudo or `--user` argument.

Reply:
in master, we just do pip install…

Reply:
Good!

Reply:
there we are ;) trying sudo now. — also, I’m gonna split the commits once this is working

Reply:
There could be another gotcha. Your motivation for this fix was build failure due to docs warning about template name change, right? Blind `pip install sphinx` will bring you latest version which may trigger more warnings due to new changes (IIRC, there was such fight for couchdb-documentation repo). If this will happens, try to install minimal required version to get the new stylesheet - 1.3.3, for instance - or any else 1.3.x. This should be enough to bypass this issue.